bazel_tools/proto.bzl

```diff
@@ -220,9 +220,11 @@ def proto_jars(
         name = "%s_jar" % name,
         srcs = None,
         deps = None,
+        runtime_deps = ["%s_jar" % label for label in proto_deps],
         resources = srcs,
         resource_strip_prefix = "%s/%s/" % (native.package_name(), strip_import_prefix),
         tags = _maven_tags(maven_group, maven_artifact_prefix, maven_artifact_proto_suffix),
+        visibility = visibility,
```

Comment:
What does that do?

Reply:
It makes these JARs visible outside the current _BUILD.bazel_ file calling `proto_jars`, which means that we can actually establish the dependencies. For example, the kvutils proto JAR needs to be able to see the DAML-LF transaction proto JAR in order to depend upon it.

Reply:
@SamirTalwar-DA Did it fail without this? IIRC the release script collects these targets via `bazel build` instead of dependencies declared in BUILD files. So, the visibility shouldn't matter.

Reply:
Oh, it's added to `runtime_deps`. Nvm, makes sense.

Reply:
@aherrmann-da: Yes; see the above comment. Bazel itself has to resolve the `runtime_deps`, so they need to see each other.